rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/WorkspaceInfo.rsc

```diff
@@ -237,12 +237,20 @@ set[loc] rascalGetOverloadedDefs(WorkspaceInfo ws, set[loc] defs, MayOverloadFun
       o scopeDefs                  // 3. Find definitions in the reached scope, and definitions within those definitions (transitively)
       ;
 
-    rel[loc from, loc to] defPaths = {};
+    rel[loc from, loc to] defPaths = fromDefPaths;
```

Comment:
This seems to fix the `constructorId` branch as to not always leave `defPaths` empty. Did you fix something for which we did not have a test yet?

Reply:
I noticed that as well. I'll think about this and see if I can come up with a test that would have before this change.

Reply:
Well, constructors were barely tested, and as you suspected, renaming them did not work as it should. Thanks for uncovering this. This led to an even more invasive rewrite of this overload detection code and adding many tests.